Carwash/order_details/tests/tests_od.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+
+from django.test import TestCase
+
+from order_details.tests.factories import AccountsFactory, UsersFactory, \
+                                          PlacesFactory, CarWashesFactory, \
+                                          ServiceFactory, OrderFactory, \
+                                          CWPlacesFactory, OrderDetailFactory
+
+from order_details.models import OrderDetail
+
+
+class OrderDetailTestCase(TestCase):
+
+    def setUp(self) -> None:
+        self.account = AccountsFactory()
+        self.user = UsersFactory(account=self.account)
+        self.place = PlacesFactory()
+        self.carwash = CarWashesFactory()
+        self.service = ServiceFactory()
+        self.order = OrderFactory(car_wash=self.carwash, user=self.user)
+        self.CWPlace = CWPlacesFactory(place=self.place, carwash=self.carwash)
+
+    def test_crud_orderdetail(self):
+
+        OrderDetailFactory(order=self.order,
+                           service=self.service,
+                           carwash_place=self.CWPlace,
+                           price=10.5,
+                           duration=15,
+                           start_time=datetime.time(18, 0, 0))
+
+        self.assertTrue(OrderDetail.objects.get(price=10.5))
```

Comment:
```suggestion
        self.assertTrue(OrderDetail.objects.filter(price=10.5).exists())
```
but in this test, you check creation factory, send request